pigpen-cascading/src/main/java/pigpen/cascading/OperationUtil.java

```diff
@@ -13,6 +13,8 @@
 
 public class OperationUtil {
 
+  public static final String SENTINEL_VALUE = "39c63d213f5cba3c7";
```

Comment:
I didn't put much thought into it. The thing is that it is easier to use a type that Cascading knows how to serialize natively. I could use the serialized version of a namespace qualified keyword though. What would be the pros and cons?